src/models/DeploymentUpdate.ts

```diff
@@ -8,16 +8,38 @@ type Base = {
   tags?: string[] | null,
   workQueueName?: string | null,
   workPoolName?: string | null,
+  infrastructureOverrides?: Record<string, unknown> | null,
+}
+
+type BaseEdit = {
+  description?: string | null,
+  schedule?: Schedule | null,
+  isScheduleActive?: boolean,
+  tags?: string[] | null,
+  workQueueName?: string | null,
+  workPoolName?: string | null,
+  infrastructureOverrides?: string,
```

Comment:
Why does this need to be a string? I thought it was being parsed back into an object when submitting the form.

Reply:
That's the model for the deployment within the form only - so for when we are editing the deployment.  It does get parsed back and we use the deploymentUpdate model everywhere else.

Reply:
ah in the component. Got it. To cut down on the duplication I'd suggest

```typescript
type BaseEdit = Omit<Base, 'infrastructureOverrides'> & {
  infrastructureOverrides?: string
}
```

Reply:
Nice!